docs/_site/index.html

```diff
@@ -16,10 +16,10 @@
 <meta property="og:url" content="http://localhost:4000/cellxgene/" />
 <meta property="og:site_name" content="cellxgene" />
 <script type="application/ld+json">
-{"description":"An interactive explorer for single-cell transcriptomics data","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/cellxgene/cellxgene-logo.png"}},"@type":"WebSite","headline":"Index","url":"http://localhost:4000/cellxgene/","name":"cellxgene","@context":"http://schema.org"}</script>
+{"publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"http://localhost:4000/cellxgene/cellxgene-logo.png"}},"description":"An interactive explorer for single-cell transcriptomics data","@type":"WebSite","url":"http://localhost:4000/cellxgene/","headline":"Index","name":"cellxgene","@context":"https://schema.org"}</script>
```

Comment:
I realize this is not due to your recent changes, but these meta tags are all wrong (see the URLs).  We should either remove or fix.

If trivial, can you fix.  If not, file a bug.

Reply:
Well spotted. I think I've fixed this with a change to the `_config.yml` file.